<launch>
	<arg name="ip" default="127.0.0.1"/>
	<arg name="port" default="5006"/>
	<node pkg="bitbots_live_tool_rqt" type="live_tool_udp.py" name="live_tool" output="screen">
		<rosparam command="load" file="$(find bitbots_live_tool_rqt)/config/live_rqt_settings.yaml"/>
		<param name="rqt_ip" value="$(arg ip)"/>
		<param name="live_rqt_port" value="$(arg port)"/>
	</node>
</launch>
